bin/lint.ts

```diff
@@ -33,19 +32,11 @@ runAll([scripts.shift()], options) // run first synchronously to install TypeScr
     if (error.results) {
       const results: Array<{ name: string; code: number }> = error.results;
       const failedType = results.find(result => result.code === 1);
+
       if (failedType) {
-        const typeDir = basename(failedType.name);
-        console.log('JSON API definitions:');
-        console.log(
-          readFileSync(join(tmpDirPath, `${typeDir}.json`)).toString()
-        );
-        console.log('Generated index.d.ts:');
-        console.log(readFileSync(join(path, typeDir, 'index.d.ts')).toString());
-        console.log('Generated tests:');
-        console.log(
-          readFileSync(join(path, typeDir, `${typeDir}-tests.ts`)).toString()
-        );
+        console.log(`::set-env name=FAILED_TYPE::${basename(failedType.name)}`);
```

Comment:
This sets the `FAILED_TYPE` environment variable to the specific type that failed, enabling us to reference it in subsequent steps.

_See [Setting an environment variable](https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable)._